pkgs/top-level/aliases.nix

```diff
@@ -818,10 +818,19 @@ mapAliases ({
   telepathy_salut = telepathy-salut; # added 2018-02-25
   telnet = inetutils; # added 2018-05-15
   terraform_1_0_0 = terraform_1_0; # added 2021-06-15
-  terraform_0_12 = throw "terraform 0.12 has been removed"; # added 2021-06-15
-  terraform_0_13 = throw "terraform 0.13 has been removed"; # added 2021-06-15
-  terraform_0_14 = throw "terraform 0.14 has been removed"; # added 2021-06-15
-  terraform_0_15 = throw "terraform 0.15 has been removed"; # added 2021-06-15
+  terraform_0_12 = throw ''
```

Comment:
Why do we have two different flavors of throws here? Why shouldn't a user of 0.14 receive the nice error message?

Reply:
I thought they were all nice... 0.14 users don't need to go by way of 0.13, so I didn't think needed the more in-depth explanation of how to go from 0.12 to 0.13.